libxkbfile-amzn2-aarch64/meta.yaml

```diff
@@ -13,30 +13,18 @@ source:
     folder: source
 
 build:
-  number: 5
+  number: 3
```

Comment:
should not be 6 ?

Reply:
It was never uploaded, and we should try to match the rpm build number from what I saw in the other recipes.